apps/api/src/app/subscribers/subscribers.controller.ts

```diff
@@ -385,8 +385,7 @@ export class SubscribersController {
     @Query('feedIdentifier') feedId: string[] | string,
     @Query('seen') seen: boolean,
     @Param('subscriberId') subscriberId: string,
-    // todo NV-2161 update DefaultValuePipe to 100 in version 0.16
-    @Query('limit', new DefaultValuePipe(1000), new LimitPipe(1, 1000, true)) limit: number
+    @Query('limit', new DefaultValuePipe(100), new LimitPipe(1, 100, true)) limit: number
```

Comment:
I am thinking about maybe it will be a good idea to remove all those LimitPipes on the count endpoint (unsee/unread etc as well) so that the user will have the ability to query the count if needed.

I saw that the notification center is using '/widgets/notifications/unseen' endpoint in useUnseenCount, so maybe we could still use the default for the users that use custom UI but i am not sure about the max limit there anymore.

wdyt?

Reply:
I agree with you :)

Reply:
I think this is a matter of the endpoint being copied from the widgets controller